python/nav/statemon/db.py

```diff
@@ -47,6 +47,11 @@
 
 _logger = logging.getLogger(__name__)
 
+# The event model requires a value severity value, even though the event engine will
```

Comment:
Typo? s/value severity/valid severity/ I assume.

Reply:
Indeed, thanks for catching it. I'll force-push a fix while merging